tests/e2e/kubetest2-kops/deployer/dumplogs.go

```diff
@@ -43,6 +44,17 @@ func (d *deployer) DumpClusterLogs() error {
 		"--private-key", d.SSHPrivateKeyPath,
 		"--ssh-user", d.SSHUser,
 	}
+	version, err := kops.GetVersion(d.KopsBinaryPath)
+	if err != nil {
+		return err
+	}
+	if version > "1.29" {
+		// we don't want to dump from more than 10 nodes
+		// kops has logic to dump the control plane instances first
+		// remove this code once kops 1.28 is EoL
+		args = append(args, "--max-nodes", "10")
```

Comment:
It takes 30 minutes to dump logs from 500 nodes. We only need logs from the control plane and few instances.

https://prow.k8s.io/view/gs/kubernetes-jenkins/pr-logs/pull/kops/16181/presubmit-kops-gce-small-scale-calico-using-cl2/1740355086892142592

Reply:
Please add this flag only for the scale tests. In other tests, we are fin with dumping more.